Extension/src/LanguageServer/Providers/codeActionProvider.ts

```diff
@@ -218,7 +226,47 @@ export class CodeActionProvider implements vscode.CodeActionProvider {
                 kind: codeActionKind
             };
             resultCodeActions.push(vscodeCodeAction);
-        });
+        };
+
+        response.commands.forEach(processCommand);
+
+        // If the refactor.inline.macro code action is specifically invoked by the user,
+        // then force a hover to ensure that the "Expands to" info is available.
+        if (!hasInlineMacro && context.only?.value === "refactor.inline.macro") {
+            const processInlineMacro = async (): Promise<boolean> => {
+                const editor: vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+                if (!editor) {
+                    return false;
+                }
+                const result: vscode.Hover[] = <vscode.Hover[]>(await vscode.commands.executeCommand('vscode.executeHoverProvider', document.uri, range.start));
+                if (result.length === 0) {
+                    return false;
+                }
+                const hoverResult: vscode.MarkdownString = <vscode.MarkdownString>result[0].contents[0];
+                if (!hoverResult.value.includes(localize("expands_to", "Expands to:"))) {
+                    return false;
+                }
+                response = await this.client.languageClient.sendRequest(GetCodeActionsRequest, params, token);
+                if (token.isCancellationRequested || response.commands === undefined) {
+                    return false;
+                }
+                for (const command of response.commands) {
+                    if (command.edit) {
+                        processCommand(command);
+                        return true;
+                    }
+                }
+                return false;
+            };
+            if (!await processInlineMacro()) {
+                const disabledCodeAction: vscode.CodeAction = {
+                    title: localize({ key: "inline_macro", comment: ["'Inline' is a command and not an adjective, i.e. like 'Expand macro'."] }, "Inline macro"),
```

Comment:
`inline.macro`

Reply:
I ran a regex to find other cases of _ , i.e. looks like I had used that for some code analysis localize calls as well, so I'll fix those too.

Reply:
Changing all the existing ones right now might reset all of the localization for them.  Maybe wait to do that until after 1.17 ships?

Reply:
@Colengms Do you know when we'll have translations by?

Reply:
I undid the other loc changes.